packages/app/src/SearchPage.tsx

```diff
@@ -645,7 +645,7 @@ export default function SearchPage() {
   }, [setResultsMode]);
 
   return (
-    <div className="LogViewerPage d-flex" style={{ height: '100vh' }}>
```

Comment:
Testing out locally I suspect this is breaking the search table scroll behavior (we're scrolling the page instead of the search results)

Reply:
oo good catch

Reply:
fixed